egs2/chime4/enh1/local/simu_ext_chime4_data_prep.sh

```diff
@@ -67,6 +73,30 @@ if [[ "$track" == "1" ]]; then
     sed -E "s#\.Clean\.wav#\.Noise\.wav#g" ${x}_spk1_wav.scp > ${x}_noise_wav.scp
   done
 
+elif [[ "$track" == "2" ]]; then
```

Comment:
Similar to the above. Can we merge 2-ch and 6-ch track?

Reply:
No, actually only the 1ch and 2ch tracks provide the audio list, while the 6ch track does not. So we have to use different logic to prepare the data.